corehq/apps/accounting/models.py

```diff
@@ -2957,24 +2958,24 @@ def create_record(cls, payment_method, transaction_id, amount):
         )
 
 
-class CreditAdjustment(models.Model):
+class CreditAdjustment(ValidateModelMixin, models.Model):
     """
     A record of any additions (positive amounts) or deductions (negative amounts) that contributed to the
     current balance of the associated CreditLine.
     """
     credit_line = models.ForeignKey(CreditLine, on_delete=models.PROTECT)
     reason = models.CharField(max_length=25, default=CreditAdjustmentReason.MANUAL,
                               choices=CreditAdjustmentReason.CHOICES)
-    note = models.TextField()
+    note = models.TextField(blank=True)
     amount = models.DecimalField(default=Decimal('0.0000'), max_digits=10, decimal_places=4)
-    line_item = models.ForeignKey(LineItem, on_delete=models.PROTECT, null=True)
-    invoice = models.ForeignKey(Invoice, on_delete=models.PROTECT, null=True)
+    line_item = models.ForeignKey(LineItem, on_delete=models.PROTECT, null=True, blank=True)
+    invoice = models.ForeignKey(Invoice, on_delete=models.PROTECT, null=True, blank=True)
     payment_record = models.ForeignKey(PaymentRecord,
-                                       on_delete=models.PROTECT, null=True)
+                                       on_delete=models.PROTECT, null=True, blank=True)
     related_credit = models.ForeignKey(CreditLine, on_delete=models.PROTECT,
-                                       null=True, related_name='creditadjustment_related')
+                                       null=True, blank=True, related_name='creditadjustment_related')
     date_created = models.DateTimeField(auto_now_add=True)
-    web_user = models.CharField(max_length=80, null=True)
+    web_user = models.CharField(max_length=80, null=True, blank=True)
```

Comment:
The Django docs recommend against having both `null` and `blank` set to `True`, https://docs.djangoproject.com/en/1.9/ref/models/fields/#null.  Seems reasonable if `None` means one thing and `""` means another, but I think that's usually not the case.  I'm guessing for the other entries this is a validation vs storage thing?